src/blockrelay/graphene_set.h

```diff
@@ -33,6 +33,7 @@ const float IBLT_DEFAULT_OVERHEAD = 1.5;
 class CGrapheneSet
 {
 private:
+    mutable uint64_t shorttxidk0, shorttxidk1;
     bool ordered;
```

Comment:
this mutable seems to have returned...did you correctly rebase this PR on top of the previous one ?

Reply:
Shoot, that must be the case; will fix. The rebase has been absolutely brutal since this is based off of #1551 and they change so many of the same things (~10 rounds of merge errors with 10s of chunks in multiple files each round). I might wait until #1551 is merged so I only have to rebase against its changes one more time.